### PR TITLE
Fix json properties

### PR DESCRIPTION
**Pull request by shargon**
Close https://github.com/neo-project/neo-modules/issues/592 https://github.com/neo-project/neo-modules/issues/594

### CODE REVIEW

**Comment by erikzhang** [comment]
Can you check if there are other places that need to be updated?

File: src/RpcClient/Models/RpcNativeContract.cs [hunk 1]
```diff
@@ -20,7 +21,7 @@ public static RpcNativeContract FromJson(JObject json)
                 Hash = UInt160.Parse(json["hash"].AsString()),
                 Nef = RpcNefFile.FromJson(json["nef"]),
                 Manifest = ContractManifest.FromJson(json["manifest"]),
-                ActiveBlockIndex = (uint)(json["activeblockindex"]?.AsNumber() ?? 0)
+                UpdateHistory = json["updatehistory"]?.GetArray().Select(u => (uint)u.GetInt32()).ToArray() ?? null
```
Comment (shargon) [inline src/RpcClient/Models/RpcNativeContract.cs:24]: @erikzhang I located this and `TransactionAttributeFromJson` issues

**Comment by shargon** [comment]
@superboyiii tested?

File: src/RpcClient/Models/RpcNativeContract.cs (hunk 1 above)
Comment (erikzhang) [inline src/RpcClient/Models/RpcNativeContract.cs:24]: I think it can't be `null`.

Comment (shargon) [reply src/RpcClient/Models/RpcNativeContract.cs]: Done

File: src/RpcClient/Utility.cs [hunk 2]
```diff
@@ -179,19 +179,25 @@ public static Signer SignerFromJson(JObject json, ProtocolSettings protocolSetti
...
         public static TransactionAttribute TransactionAttributeFromJson(JObject json)
         {
             TransactionAttributeType usage = Enum.Parse<TransactionAttributeType>(json["type"].AsString());
-
-            switch (usage)
+            return usage switch
             {
-                default: throw new FormatException();
-            }
+                TransactionAttributeType.HighPriority => new HighPriorityAttribute(),
```
Comment (ProDog) [inline src/RpcClient/Utility.cs:192]: Can add UT for this section?

File: tests/Neo.Network.RPC.Tests/RpcTestCases.json [hunk 3]
```diff
@@ -1194,7 +1194,7 @@
             "trusts": [],
             "extra": null
           },
-          "activeblockindex": 0
+          "updatehistory": null
```
Comment (ProDog) [inline tests/Neo.Network.RPC.Tests/RpcTestCases.json:1197]:
```suggestion
          "updatehistory": [0]
```

File: tests/Neo.Network.RPC.Tests/RpcTestCases.json [hunk 4]
```diff
@@ -1194,7 +1194,7 @@
             "trusts": [],
             "extra": null
           },
-          "updatehistory": null
+          "updatehistory": [0]
```
Comment (ProDog) [inline tests/Neo.Network.RPC.Tests/RpcTestCases.json:1197]: Need to modify all `updatehistory`😂

**Comment by ProDog** [comment]
Tested OK.

**Review by superboyiii — APPROVED** [review]

**Comment by superboyiii** [comment]
Merge?

**Review by ProDog — APPROVED** [review]

**Review by erikzhang — APPROVED** [review]

**Pull request merged**